big_tests/tests/muc_http_api_SUITE.erl

```diff
@@ -52,6 +54,9 @@ complex() ->
      multiparty_multiprotocol
     ].
 
+failure_response() ->
+    [failed_invites,
```

Comment:
AFAIR, we don't have invites mechanism, only affiliation change requests. so this name can be very confusing, especially when we finally implement invites.

Reply:
sorry, I didn't catch that it's about a classic MUC